db/migrate/20161128200400_update_quotes.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateQuotes < ActiveRecord::Migration
+  def change
+    change_column_null :quotes, :anime_id, false
+    change_column_null :quotes, :user_id, false
+    change_column_null :quotes, :content, false
+
+    add_foreign_key :quotes, :users
+    add_foreign_key :quotes, :anime
+    add_foreign_key :quotes, :characters
```

Comment:
You're right, if we leave setting a database default, it will just error out with a _not null constraint_, which is more beneficial to us than assigning the quote to the wrong media type.

Thanks for pointing that out 😅